src/runtime_src/core/edge/drm/zocl/zocl_drv.c

```diff
@@ -170,7 +170,7 @@ static int zocl_pr_slot_init(struct drm_zocl_dev *zdev,
 
 		zocl_slot->partial_overlay_id = -1;
 		zocl_slot->slot_idx = i;
-		zocl_slot->slot_type = DOMAIN_PL;
+		zocl_slot->slot_type = ZOCL_PL_ONLY_XCLBIN;
```

Comment:
DOMAIN_PL and ZOCL_PL_ONLY_XCLBINs are differrent. I didnt understand how u r mapping this.